src/grpc.ts

```diff
@@ -122,20 +117,17 @@ export class GrpcClient {
       this.grpc = options.grpc!;
       this.grpcVersion = '';
     } else {
-      // EXPERIMENTAL: If GOOGLE_CLOUD_USE_GRPC_JS is set, use the JS-based
-      // implementation of the gRPC client instead. Requires http2 (Node 8+).
-      if (
-        semver.gte(process.version, '8.13.0') &&
-        !!process.env.GOOGLE_CLOUD_USE_GRPC_JS
-      ) {
-        this.grpc = require('@grpc/grpc-js');
+      if (semver.gte(process.version, '8.13.0')) {
+        this.grpc = grpc;
         this.grpcVersion = require('@grpc/grpc-js/package.json').version;
       } else {
-        this.grpc = require('grpc');
-        this.grpcVersion = require('grpc/package.json').version;
+        const errorMessage =
+          'To use @grpc/grpc-js you must run your code on Node.js v8.13.0 or newer. Please see README if you need to use an older version. ' +
+          'https://github.com/googleapis/gax-nodejs/blob/master/README.md';
+        console.error(errorMessage);
```

Comment:
I don't think we need to write to the console if we're throwing, do we?

Reply:
I just wanted to communicate the error in all the ways possible :)

Reply:
I agree with Justin, throw is probably sufficient, as it will end up in `stderr` and output essentially the same message (but with stack trace).

Reply:
Removed `console.error`.